### ISSUE TITLE
[NPC] Elwynn Forest - Marshal Dughan

### ISSUE THREAD

**Issue by hugomarciano** [HellscreamWoW/Tracker#2583]
NPC: Marshal Dughan
Location: Elwynn Forest

Problem: "The Marshal Dughan should lend me a horse to take me to the Guard Thomas is on the eastern side of Elwynn Forest. But it does not have this option."

**Comment by Guybrush-Hellscream** [comment]
I believe this is for the quest [Further Concerns](http://www.wowhead.com/quest=35/further-concerns).

**Comment by hugomarciano** [comment]
It is not a prerequisite for this quest. @Guybrush-Hellscream

**Comment by Guybrush-Hellscream** [comment]
Yeah i know, but the quest is a prerequisite to the travel. You can't travel with his horse without having this quest.

**Comment by Tommy150** [comment]
**[QUEST] Further Concerns**

When the player accepts the quest, the quest giver, Marshal Dughan, should offer them a horse to Guard Thomas.
The horse should be uncontrollable.

_Here are some WoWHead links related to the quest:_
QUEST: http://www.wowhead.com/quest=35/further-concerns
START: http://www.wowhead.com/npc=240/marshal-dughan
NPC: http://www.wowhead.com/npc=42260/stormwind-charger
END: http://www.wowhead.com/npc=261/guard-thomas

_Here is a video recording of the quest:_
https://www.youtube.com/watch?v=QcMiaZ0FxAs

I hope the information are going to be usefull and the quest is going to be fixed soon.
